op-dispute-mon/config/config.go

```diff
@@ -0,0 +1,55 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+
+	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum-optimism/optimism/op-service/oppprof"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr"
+)
+
+var (
+	ErrMissingL1EthRPC = errors.New("missing l1 eth rpc url")
+)
+
+const (
+	DefaultMaxPendingTx = 10
+)
+
+// Config is a well typed config that is parsed from the CLI params.
+// It also contains config options for auxiliary services.
+type Config struct {
+	L1EthRpc     string // L1 RPC Url
+	MaxPendingTx uint64 // Maximum number of pending transactions (0 == no limit)
+
+	TxMgrConfig   txmgr.CLIConfig
```

Comment:
We don't need a tx mgr in the monitor. It's purely read only.

Reply:
Ah this was an oversight on my part, thank you. Removing.